### ISSUE TITLE
Turning on sidebar

### ISSUE THREAD

**Issue by amarbrkic** [zzossig/hugo-theme-zzo#441]
Is there way to turn on sidebar when viewing posts??

**Comment by szpak** [comment]
Not really: https://github.com/zzossig/hugo-theme-zzo/issues/221#issuecomment-596270553

**Comment by mamanongnong** [comment]
The following error occurred while running the Hugo Server command.
The error is as follows:
----------------------------------------------------------------------
$ hugo server --themesDir ../..
Start building sites …
hugo v0.88.1-5BC54738+extended windows/amd64 BuildDate=2021-09-04T09:39:19Z VendorInfo=gohugoio
Error: Error building site: "D:\myhugo\mydocs\themes\zdoc\exampleSite\content\en\posts\rich-content.md:1:1": timed out initializing value. You may have a circular loop in a shortcode, or your site may have resources that take longer to build than the `timeout` limit in your Hugo config file.
Built in 14064 ms
-------------------------------------------------------------------------
Is there something wrong with my configuration? Or there is a problem with the Hugo theme that needs to be fixed
Please give me a reply when received. Thank you 🙂
